web/css/luna/licensetoplay.css

```diff
@@ -0,0 +1 @@
+article>div{color:#8a78a8}
```

Comment:
There is no article on the page anymore. This css affects nothing, we should find the less line that causes this to be generated and correct it.